src/core/css/SearchResult.scss

```diff
@@ -73,11 +81,4 @@
 .SearchResult-actions {
   align-items: center;
   display: flex;
-}
-
-.SearchResult-icon {
-  display: inline-block;
-  float: right;
-  height: 38px;
-  width: 38px;
-}
+}
```

Comment:
Super nitpick here but we like to have newlines at the end of files; often you can get your text editor to add these automatically when saving. Helps to avoid wondering if a file prematurely ends 😄